backend/src/main/java/de/cae/XYFleet/ressource/BookingRessource.java

```diff
@@ -35,17 +32,16 @@ protected void doInit() throws ResourceException {
     }
 
     @Delete()
-    public String deleteBooking() {
+    public String deleteBooking() throws ResourceException {
         checkInRole(ROLE_USER);
 
         String result = this.toString();
 
+
+        if (Integer.parseInt(getClientInfo().getUser().getIdentifier()) != bookingIdentifier && !isInRole("admin"))
+            throw new ResourceException(403);
         //DELETE bookings where id = {bookingIdentifier}
-        if(Integer.parseInt(getClientInfo().getUser().getName()) == bookingIdentifier  || isInRole("admin")){
-            dslContext.delete(BOOKINGS).where(BOOKINGS.ID.eq(bookingIdentifier)).execute();
-        }else{
```

Comment:
Exhibit B

Reply:
What do you mean by eshibit B? Is it a way to tell me that it is the same Issue like Exhibit A?

Reply:
Exhibit B is the second example for unformated code.

Reply:
> try reformating the whole project (or at least your changes) with your IDEs standards for consistency' sake. See the Exhibits (there are way more, I just gave up)